gems/pending/openstack/amqp/openstack_ceilometer_event_monitor.rb

```diff
@@ -0,0 +1,76 @@
+require 'openstack/openstack_event_monitor'
+require 'openstack/amqp/openstack_amqp_event'
+require 'openstack/amqp/openstack_ceilometer_event_converter'
+
+class OpenstackCeilometerEventMonitor < OpenstackEventMonitor
+  def self.available?(_options = {})
+    # TODO(maufart): add lookup for ceilometer_enabled in vmdb config (from MiqRunner?)
+    true
```

Comment:
@blomquisg are you ok, for the backported version, that this will be figured out automatically?

So first we will try if amqp is there, next we will try ceilometer?